sentry-android-timber/proguard-rules.pro

```diff
@@ -2,4 +2,8 @@
 
 -keep class io.sentry.android.timber.** { *; }
 
+# To ensure that stack traces is unambiguous
+# https://developer.android.com/studio/build/shrink-code#decode-stack-trace
+-keepattributes LineNumberTable,SourceFile
```

Comment:
Doesn't this get through because of `android-core` already or do we need to specify it here also?

Reply:
written in the PR template

> rules are per module, so some of them need to be placed in all of our android modules.